packages/blitz-paint/src/render.rs

```diff
@@ -395,6 +400,11 @@ struct ElementCx<'a> {
     devtools: &'a DevtoolSettings,
 }
 
+///TODO: If BoundingBox implements `Shape`, remove this. This exists to side-step it not currently doing that.
+fn convert_rect(rect: &parley::BoundingBox) -> peniko::kurbo::Rect {
+    peniko::kurbo::Rect::new(rect.x0, rect.y0, rect.x1, rect.y1)
```

Comment:
No need to reference `kurbo` via `peniko` here. `blitz-paint` depends on `kurbo` directly.